govcd/edgegateway_lb_service_monitor_test.go

```diff
@@ -0,0 +1,69 @@
+// +build lb lbsm functional ALL
```

Comment:
"lb" is clear, but "lbsm" is too much for my taste :) Maybe let's use full lb + full state monitor words?
CC: @dataclouder